support/uitls.go

```diff
@@ -220,12 +212,39 @@ func ConvertECAddressToUser(addr []byte) string {
 	return base58.Encode(userd)
 }
 
-
 // Convert a User facing Factoid or Entry Credit address
 // or their Private Key representations
 // to the regular form.  Note validation must be done
 // separately!
-func ConvertFctECUserStrToAddress(userFAddr string) string {
+func ConvertUserStrFctEcToAddress(userFAddr string) (string, error) {
```

Comment:
The factom lib has this, and it checks the checksum too:
https://github.com/FactomProject/factom/blob/master/addresses.go#L43